portfolio/src/main/webapp/style.css

```diff
@@ -14,101 +16,310 @@
     margin-right: auto;
     width: 50%;
 }
-body {
-    background-image: url('https://img.freepik.com/free-photo/white-concrete-wall_53876-92803.jpg');
-}
-h1 {text-align: center;}
+
+/* New portfolio styling */
+
 *{
```

Comment:
[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space between the last selector and the declaration block.